nixos/modules/programs/skim.nix

```diff
@@ -8,38 +8,30 @@ let
   inherit (lib)
     mkEnableOption
     mkPackageOption
-    optional
     optionalString
+    mkIf
     ;
   cfg = config.programs.skim;
 in
 {
   options = {
     programs.skim = {
-      fuzzyCompletion = mkEnableOption "fuzzy completion with skim";
```

Comment:
You should probably add a `mkRemovedOptionModule` notice to trigger on eval.

See https://github.com/NixOS/nixpkgs/blob/90807be07a580d339d0d919fb5fb488fe4dddf6f/nixos/modules/services/misc/forgejo.nix#L58 for a usage example